Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Analytic Rules/IoTPLCStopCommand.yaml

```diff
@@ -13,10 +13,12 @@ queryPeriod: 1h
 triggerOperator: gt
 triggerThreshold: 0
 tactics:
-  - Impact
+  - DefenseEvasion
```

Comment:
Can you please validate if tactics is DefenseEvasion or Evasion in this case

Reply:
It's DefenseEvasion based on Sentinel  @vakohl